assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-title-block/edit.tsx

```diff
@@ -1,23 +1,35 @@
 /**
  * External dependencies
  */
-import { useBlockProps } from '@wordpress/block-editor';
-
-/**
- * Internal dependencies
- */
-import Block from './block';
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
 
 export const Edit = (): JSX.Element => {
-	const blockProps = useBlockProps();
+	const blockProps = useBlockProps( {
+		className: 'wc-block-mini-cart__title',
+	} );
+
+	const TEMPLATE = [
+		[ 'woocommerce/mini-cart-title-label-block', {} ],
+		[ 'woocommerce/mini-cart-title-items-counter-block', {} ],
+	];
 
 	return (
-		<div { ...blockProps }>
-			<Block />
-		</div>
+		<h2 { ...blockProps }>
+			<InnerBlocks
+				allowedBlocks={ [
+					'woocommerce/mini-cart-title-label-block',
+					'woocommerce/mini-cart-title-items-counter-block',
+				] }
+				template={ TEMPLATE }
```

Comment:
For some reason I'm still able to move the blocks. :confused:  I noticed that setting `templateLock="all"`, fixes it.

Reply:
Done!

Reply:
I can't see this change. Maybe you forgot to push? (Or I'm missing something :sweat_smile: )

Reply:
Sorry, completely missed this comment, my `Done!` was meant for the comment above this one 🤦 
Now I pushed 😅